rdt-open-reader/src/main/java/io/ona/rdt/activity/PatientRegisterActivity.java

```diff
@@ -67,6 +72,8 @@ protected void onCreate(Bundle savedInstanceState) {
         NavigationView navigationView = findViewById(R.id.nav_view);
         setupDrawerContent(navigationView);
         requestPermissions();
+
+        LocalBroadcastManager.getInstance(this).registerReceiver(latestSyncDateReceiver, new IntentFilter(ACTION_UPDATE_LATEST_SYNC_DATE));
```

Comment:
We can also simply use whatever broadcast is used on successful sync instead of declaring a custom new one.

Reply:
If we will do that then the `updateSyncDate` method will be called unnecessary even we don't need to call this.

Reply:
Do we not need to call it after every successful sync?

Reply:
yes, because we need to update the label after a successful sync.

Reply:
I think what I'm trying to say is: there's a broadcast sent by client core when sync is successful. Instead of using our own custom one, simply register a receiver here that listens to that broadcast and call `updateSyncDate`.